test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,333 @@
+-   Check if `Breadcrumbs` renders navigation up button
+
+Expected look:  
+![BreadcrumbsRoot](../resources/group-breadcrumbs-root.png)
+  
+![BreadcrumbsNested](../resources/group-breadcrumbs-nested.png)
+
+--------------------------------------------------------------------------------
+
+##### TC014
+
+### Navigation up works as expected
+
+-   Add a `Group` block
+-   Create nested structure
+-   Select deep nested block
+-   Press navigation up button on `Breadcrumb`
+-   Check if each time you press navigation up button the selection moves to parent block
+-   Check if border and dim styles changes accordingly
+
+--------------------------------------------------------------------------------
+
+##### TC015
+
+### Navigation down works as expected
+
+-   Add a `Group` block
+-   Create some nested structure ( at least 3 levels deep )
+-   Clear selection
+-   Press on the bottom-most block in hierarchy of added `Group`
+-   Check if the selection is redirect properly to top-most parent
+-   Repeat - press on the bottom-most block in hierarchy of added `Group`
+-   Check if each time you press nested block you move selection one level down (block which gets selection should be common ancestor with previously selected block and pressed block)
+
+--------------------------------------------------------------------------------
+
+##### TC016
+
+### Navigation in hierarchy works as expected
+
+-   Add a `Group` block
+-   Create some nested structure ( at least 3 levels deep )
+-   Select nested block
+-   Having nested block selected try to select block which is higher in the hierarchy (one, two and more levels above)
+-   Check if that block gets selected after press
+
+--------------------------------------------------------------------------------
+
+##### TC017
+
+### Mover arrows changes the order of blocks in group
+
+-   Add a `Group` block
+-   Create nested structure
+-   Check if using mover arrows change the order of blocks in `Group` as expected 
+
+--------------------------------------------------------------------------------
+
+##### TC018
+
+### Delete button removes the block from group and group block
+
+-   Add a `Group` block
+-   Create nested structure
+-   Selecting nested blocks
+-   Check if `Delete` button removes the selected block
+ 
+-   Select `Group`
+-   Check if `Delete` button removes the whole `Group` and it's children
+
+--------------------------------------------------------------------------------
+
+##### TC019
+
+### Ungroup button works as expected
+
+-   Add a `Group` block
+-   Add blocks to `Group`
+-   Select `Group` block
+-   Press `Ungroup` button on the `MobileToolbar`
+-   Check if `Group` block was delete and all of it's child are able to select directly after press
+
+--------------------------------------------------------------------------------
+
+##### TC020
+
+### New block added to group gets focus immedately
```

Comment:
we can remove this, already a standard behavior

Reply:
don